### DISCUSSION TITLE
The explorer is in UNPUBLISHED status

### DISCUSSION THREAD

**Discussion by ansfm** [WalletConnect/walletconnect-monorepo#5168] [Developer Support] [answered]
Our wallet was approved before but right now I found it has "UNPUBLISHED" status and not able to find it on the search anymore. I checked the explorer dashboard and I can't find anywhere to resubmit or do something to make it available again. The project ID is 61537a0696f06e0338455e9d8a58fdbd. Thanks for your help!

**Comment by glitch-txs** [comment]
I can see it as published as well as in the modal

**Reply by ansfm** [reply]
Unfortunately, It is "unpublished" and I can't find it on the modal as you mentioned. Please check my screenshot for its status.
<img width="704" alt="Screenshot 2024-07-09 at 14 32 28" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/171531136/a6b14f34-475a-4abd-8936-1a024813b5d5">

**Reply by ansfm** [reply]
@glitch-txs Could you please tell me where you saw it or do you need further information?

**Reply by glitch-txs** [reply, marked as answer]
mmm it's showing up now, not sure what happened, I've republish it, we'll check on this 👍

**Comment by ansfm** [comment]
@glitch-txs This is not original issue but I made an update for the Explorer about a week ago. Could you please help to review it?
